doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,421 @@
+zorlsfc,Surface roughness
+vfracsfc,Vegetation fraction
+wetness,Soil moisture availability in top soil layer
+nirbmdi,Surface near IR beam shortwave downward flux
+nirdfdi,Surface near IR differential shortwave downward flux
+visbmdi,Surface uv+visible beam shortwave downward flux
+visdfdi,Surface uv+visible differential shortwave downward flux
+xlaixy,Leaf area index
+vfrac,Fraction of vegetation category
+sfrac,Fraction of soil category
+slc,Liquid soil moisture
+soill,Liquid soil moisture
+soilw,Volumetric soil moisture
+soilt,Soil temperature
+tref,NSST reference or foundation temperature
+z_c,NSST sub-layer cooling thickness
+c_0,NSST coefficient1 to calculate d(tz)/d(ts)
+c_d,NSST coefficient2 to calculate d(tz)/d(ts)
+w_0,NSST coefficient3 to calculate d(tz)/d(ts)
+w_d,NSST coefficient4 to calculate d(tz)/d(ts)
+xt,NSST heat content in diurnal thermocline layer
+xs,NSST salinity content in diurnal thermocline layer
+xu,NSST u-current content in diurnal thermocline layer
+xv,NSST v-current content in diurnal thermocline layer
+xz,NSST diurnal thermocline layer thickness
+zm,NSST mixed layer thickness
+xtts,NSST d(xt)/d(ts)
+xzts,NSST d(xt)/d(ts)
+d_conv,NSST thickness of free convection layer
+ifd,NSST index to start dtlm run or not
+dt_cool,NSST sub-layer cooling amount
+qrain,NSST sensible heat flux due to rainfall
+nwfa,Number concentration of water-friendly aerosols
+nwfa2d,Water-friendly surface aerosol source
+nifa,Number concentration of ice-friendly aerosols
+nifa2d,Ice-friendly surface aerosol source
+thompson_diag3d,Thompson extended diagnostics array
+fire_heat,Surface fire heat flux
+burned,Ration of the burnt area to the grid cell area
+emdust,Emission of fine dust for smoke
+emseas,Emission of seas for smoke
+emanoc,Emission of anoc for thompson mp
+coef_bb_dc,Coefficient bb for smoke
+min_fplume,Minimum smoke plume height
+max_fplume,Maximum smoke plume height
+HWP,Hourly fire weather potential
+HWP_ave,Averaged fire weather potential
+uspdavg,Boundary layer average wind speed
+hpbl_thetav,Boundary layer depth modified parcel method
+drydep_smoke,Dry deposition smoke
+drydep_dust,Dry deposition dust
+drydep_coarsepm,Dry deposition coarse pm
+wetdpr_smoke,Resolved wet deposition smoke
+wetdpr_dust,Resolved wet deposition smoke
+wetdpr_coarsepm,Resolved wet deposition coarse pm
+wetdpc_smoke,Convective wet deposition smoke
+wetdpc_dust,Convective wet deposition smoke
+wetdpc_coarsepm,Convective wet deposition coarse pm
+peak_hr,Hour of peak smoke emissions
+fire_type,Fire type
+lu_nofire,Lu nofire pixes
+lu_qfire,Lu qfire pixes
+fhist,Coefficient to scale the fire activity depending on the fire duration
+fire_end_hr,Hours since fire was last detected
+ebb_smoke_in,Input smoke emission
+frp_output,Output fire radiative power
+ebb_rate,Total EBB carbon emissions
+frp_davg,Daily mean fire radiative power
+hwp_davg,Daily mean hourly wildfire potential
+ebu_smoke,Smoke emission
+ext550,3D total extinction at 550nm

```

Comment:
@selbigmtnwx23 Could you add `radar_tten` (lines 5048-5065) below `ext550`? It seems to be missing from the table.